RobotFramework_Testsuites/Config/CConfig.py

```diff
@@ -469,7 +469,7 @@ def __loadConfigFileLevel2(self):
         
         oJsonPreprocessor = CJsonPreprocessor(syntax="python")
         try:
-            oSuiteConfig = oJsonPreprocessor.jsonLoad(self.sTestSuiteCfg)
+            oSuiteConfig = oJsonPreprocessor.jsonLoad(self.sCalcAbsPath(self, self.sTestSuiteCfg))
```

Comment:
[est-fullautomation](https://github.com/test-fullautomation) Pending

Hi Son,
static method should be called with CConfig instead of self.
Thank you,
Thomas